### DISCUSSION TITLE
Missing header file?

### DISCUSSION THREAD

**Discussion by fafalone** [Open-Shell/Open-Shell-Menu#1532] [Q&A] [answered]
ClassicExplorer_i.h is included by several files, and is referenced in the .vcprojx file, but is nowhere to be found. Curiously, it's also missing from the original ClassicShell repo too. I pulled some random releases, and haven't found one containing it.

So far it just appears to define some GUIDs that could be reproduced from ClassicExplorer.idl, but in my experience missing files are rarely so easily replaced. 

I feel like I might be missing something obvious here as searching turns up nothing about this despite apparently being a long-standing issue. Apologies if that's the case.

**Comment by ge0rdi** [comment, marked as answer]
That header file is generated by MIDL compiler during project compilation.
It is always generated using content of `ClassicExplorer.idl` file so there is no need to store it inside repo.

**Reply by fafalone** [reply]
I see. Thanks.
